### DISCUSSION TITLE
About GitHub Repo Clone

### DISCUSSION THREAD

**Discussion by FaultyxDx** [AndroidIDEOfficial/AndroidIDE#102] [Q&A]
Is there any way to compile git repository in Android IDE

**Comment by Smooth-E** [comment]
(use externally installed termux if described method does not  work in terminal, included with ide)
```
pkg up
pkg update
pkg install git
pkg install gh
termux-setup-storage
cd storage/shared/AndroidIDEProjects/cloned-repo
git init
git clone 
```
You should also [authorize you into git](https://cli.github.com/manual/gh_auth_login)

**Reply by itsaky** [reply]
If you're using APK from GitHub actions, then you can install git in AndroidIDE's terminal (`pkg install git`). Currently, there is not GUI interface for GIT. It is planned to be added.

As @Smooth-E said, you could also use Termux to manage git repositories.
